example/server/pushAzure.js

```diff
@@ -0,0 +1,52 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
```

Comment:
Slightly worried about putting a MS copyrighted file into the repo. I know you work for MS but I don't want to run afoul of your legal team after some automated scan of github repos. How can we show you have permissions to add this file?